app-misc/piper/files/piper-0.5.1-disable-flake8-linting.patch

```diff
@@ -0,0 +1,14 @@
+--- piper-0.5.1/meson.build	2020-06-22 01:07:53.000000000 -0700
++++ piper-0.5.1/meson.build	2021-08-08 11:46:32.984756474 -0700
+@@ -119,11 +119,3 @@
+                configuration: config_piper_devel)
+ 
+ meson.add_install_script('meson_install.sh')
+-
+-flake8 = find_program('flake8', required: false)
+-if flake8.found()
```

Comment:
If making patches that are never going to be upstreamed, you can (optionally) cheat to have shorter patches by (for example) using `if false`

But it's fine if you prefer it like that too.

Reply:
This only results in one line saved for this particular patch, however, I will keep it in mind for larger patches.